deps/buffer.lua

```diff
@@ -35,6 +35,7 @@ local instanceof = core.instanceof
 
 ffi.cdef[[
   void *malloc (size_t __size);
+  void *calloc (size_t nmemb, size_t __size);
```

Comment:
Why do the arguments here start with `__`?  Should the nmemb argument be `__nmemb` too?